### DISCUSSION TITLE
Rename repo

### DISCUSSION THREAD

**Discussion by andreivladbrg** [sablier-labs/flow#25] [General]
Since we are going to make the [Package Tethering](https://github.com/sablier-labs/v2-core/issues/808) change in `v2-core` should we also remove "V2" from this project? 

It means that we need to:
- [ ] rename the repo
- [ ] rename all contracts that include "V2"

What should be the repo name? 
- evm-open-ended
- open-ended

@smol-ninja @PaulRBerg

**Comment by PaulRBerg** [comment]
Let's hold on to renaming things here until I get a chance to review this code base properly.

I'm undecided that `open-ended` is the best name for this project, so rather than have to do multiple refactors, let's just do one.

**Reply by andreivladbrg** [reply]
> I'm undecided that open-ended is the best name for this project, so rather than have to do multiple refactors, let's just do one

got it

leaving this discussion open, tho

**Comment by andreivladbrg** [comment]
Why did you close this? Let's keep this in the backlog, since we will need to remove the "V2" naming.

**Reply by PaulRBerg** [reply]
Discussions are for discussing things, they are not todos. If you want to open an issue for this, do it, but I don't think that's needed because the refactor will be implicit in my reviewing this repository.
